hw/ip/clkmgr/data/clkmgr.hjson.tpl

```diff
@@ -394,6 +394,16 @@
           desc: '''
             ${src} has encountered a measurement error.
           '''
+        },
```

Comment:
nit: You don't have to put comma in Hjson

Reply:
ah yeah, honestly that's only there for my editor, which helps tab things correctly when the comma is there.

Reply:
Emacs user! 👍

Reply:
i really think i need to investigate a new editor hehe